.deploy/prod-gcp.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: familie-dokument
+  namespace: teamfamilie
+  labels:
+    team: teamfamilie
+
+spec:
+  image: {{ image }}
+  team: teamfamilie
+  port: 8082
+  liveness:
+    path: /familie/alene-med-barn/mellomlagring/internal/status/isAlive
```

Comment:
Not sure if we need this long context-path now, since we use subdomain for ingress and not different paths for GCP? 
On-prem: nav.no/familie/alene-med-barn/<app>
GCP: <app>.nav.no

Reply:
I'll check with people in baks. Thank you!

Reply:
Liveness is not conected to the ingress. It should be `/internal/isAlive`

Reply:
@henninghaakonsen Are you sure? This is copied from SBS settings, where it works.

Reply:
Maybe, but then it is configured wrong.

Reply:
Ok, I'll fix it later, when the deployment issue gets fixed